### DISCUSSION TITLE
Renovate PRs display the `approve and run` button

### DISCUSSION THREAD

**Discussion by Neenu1995** [renovatebot/renovate#14839] [Request Help]
Recently some of the renovate PRs opened in java client libraries in the googleapis org is displaying the `Approve and run` button.
But the checks do run fine. This is not the first time renovate bot has opened a PR in any of these repos. None of our automation has changed recently to cause this.
Example PR: https://github.com/googleapis/google-auth-library-java/pull/826
![image](https://user-images.githubusercontent.com/13271327/160501930-8ad0be70-9241-4186-a462-ea00b50ed80a.png)
 
Any idea what could be the cause?

cc: [danoscarmike](https://github.com/danoscarmike) [meltsufin](https://github.com/meltsufin) [suztomo](https://github.com/suztomo)

**Comment by rarkins** [comment]
I'm not aware of any "identity" changes for the bot so it seems most likely a GitHub glitch
